easybuild/easyconfigs/p/python-parasail/python-parasail-1.1.12-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = "PythonPackage"
+
+name = 'python-parasail'
+version = '1.1.12'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.org/project/parasail/'
+description = """ This package contains Python bindings for parasail. """
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://pypi.python.org/packages/source/p/parasail']
+sources = [{'filename': SOURCE_TAR_GZ, 'download_filename': 'parasail-%(version)s.tar.gz'}]
+checksums = ['2056359885f4ee569b2c505e8b700c09cf03d6d74f3889a6de166e92aa5c827b']
+
+builddependencies = [
+    ('parasail', '2.2'),
+    ('wheel', '0.31.0', versionsuffix),
+]
+dependencies = [
+    ('Python', '2.7.14'),
+]
+
+download_dep_fail = True
+
+# make sure setup.py finds the parasail library
+preinstallopts = "ln -s $EBROOTPARASAIL/lib/libparasail.so parasail/libparasail.so && "
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%%(pyshortver)s/site-packages/parasail'],
```

Comment:
@smoors sanity check fails because of the double `%` which escapes the template value `%(pyshortver)s`